src/openrct2/core/FileIndex.hpp

```diff
@@ -144,6 +144,9 @@ class FileIndex
         std::vector<std::string> files;
         for (const auto& directory : SearchPaths)
         {
+            if (directory.empty())
+                continue;
```

Comment:
I think I'd prefer for an empty directory to not end up in this `SaearchPaths` vector in the first place... But it's fine, I suppose.